sdk/identity/azure-identity/samples/CMakeLists.txt

```
@@ -7,6 +7,11 @@ project (azure-identity-samples LANGUAGES CXX)
 set(CMAKE_CXX_STANDARD 14)
 set(CMAKE_CXX_STANDARD_REQUIRED True)
 
+add_executable(chained_token_credential_sample chained_token_credential.cpp)
+target_link_libraries(chained_token_credential_sample PRIVATE azure-identity)
+target_include_directories(chained_token_credential_sample PRIVATE .)
+create_per_service_target_build(chained_token_credential_sample identity)
```

pull latest changes from `main` and use macro `create_per_service_target_build_for_sample` instead of this one as:

```suggestion
create_per_service_target_build_for_sample(identity chained_token_credential_sample)
```